sodetlib/operations/bias_steps.py

```diff
@@ -1159,7 +1159,8 @@ def take_bias_steps(S, cfg, bgs=None, step_voltage=0.05, step_duration=0.05,
         dc_biases = initial_dc_biases
         init_current_mode = sdl.get_current_mode_array(S)
         if high_current_mode:
-            dc_biases = dc_biases / S.high_low_current_ratio
+            dc_biases[init_current_mode == 0] = (dc_biases[init_current_mode == 0] /
+                                                 S.high_low_current_ratio)
             step_voltage /= S.high_low_current_ratio
```

Comment:
@dpdutcher this `step_voltage` should probably also get the same treatment so that the step amplitude is equal for all bias lines independent of their current mode.

Reply:
I'm not sure if step_voltage needs to be changed, because the function parameter `step_voltage` should always be in low current mode since that is user controlled

Reply:
Ahh ok @jlashner then do you think that the comment I made about the analysis function still holds then? Does bsa.high_current_mode need to be a boolean array instead of a boolean scalar?

Reply:
From what I can tell the analysis shouldn't need to be changed, but I think we should definitely test before merging. If I'm not mistaken all bias lines should still be in high current mode during the steps, so we should only need a single boolean. The change here is that if all bias lines are not in high current mode before the step, it will only correct the ones that are not.